doc/sphinx/language/core/conversion.rst

```diff
@@ -24,8 +24,8 @@ application of a function to a given argument with its result. For
 instance the identity function over a given type :math:`T` can be written
 :math:`λx:T.~x`. In any global environment :math:`E` and local context
 :math:`Γ`, we want to identify any object :math:`a` (of type
-:math:`T`) with the application :math:`((λ x:T.~x)~a)`.  We define for
-this a *reduction* (or a *conversion*) rule we call :math:`β`:
+:math:`T`) with the application :math:`((λ x:T.~x)~a)`.
+This is referred to as :gdef:`β-reduction <beta-reduction>`.
```

Comment:
Technical nit: The identification isn't called β-reduction.  The identification is undirected, while β-reduction is directed.  The identification follows from β-reduction, which is given in the mathematical equation below.

Reply:
Think I got it.  What is the meaning of "identify" in this paragraph?

Reply:
"declare to be convertible"

Reply:
How about "We want to declare some terms to be convertible so we can identify the application of a function to a given argument with its result."

And maybe change "identify" to "associate", "reduce" or "map"?

Reply:
IMHO this section is badly written. The part showing intent ("we want to declare some terms to be convertible") is useless and could be removed. Let's try to be factual instead: beta-reduction reduces a beta-redex (the application of a function to an argument) and thus allows to identify the two as convertible.

Reply:
I like @Zimmi48 's suggestion, and have a minor not: "allows to identify" should be "allows us to identify" or "allows Coq to identify" or "allows identification of".  And perhaps "function" should be replaced with "lambda" since otherwise it might refer to a term of function type, rather than a term which is syntactically headed by a lambda.

Reply:
I was thinking with Gallina in mind, where lambda is `fun`.

Reply:
I made it "function definition" rather than "lambda".  Couldn't find the old text with "allows to identify".  Let's consider further revisions of the conversion chapter in #13815.